### PR TITLE
Add mint ui example

### PR DESCRIPTION
**Pull request by MarkSackerberg**
How this example can and should be used is further described in this PR to the docs https://github.com/metaplex-foundation/docs/pull/391

The docs will explain everything, nevertheless we should allow the user to just clone the page to allow them to focus on understanding instead of copy and pasting code.

This page is based on the `connect-wallet` example

### CODE REVIEW

File: getting-started-react-cra5/src/App.js [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
+  const [candy, setCandy] = useState(null);
 
-    setNft(asset);
+  const connection = new Connection(clusterApiUrl("devnet"));
+  const wallet = useWallet();
+  const mx = Metaplex.make(connection);
+  mx.use(walletAdapterIdentity(wallet));
+  const collectionUpdateAuthority = new PublicKey("9eYgKf44KuPocpSpC3QtLqkjtMjHTbUP4KgrC7mYz4GM");
+  
+  const mintNft = async () => {
```
Comment (lorisleiva) [inline getting-started-react-cra5/src/App.js:21]: Any reason this "fetch NFT" operation was turned into a "mint NFT" operation?

File: mint-ui-example/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <WalletModalProvider>
+        <MetaplexProvider>
+          <div className={styles.App}>
+            <WalletMultiButton />
+            <MintNFTs onClusterChange={handleChange} />
+          </div>
+        </MetaplexProvider>
+        </WalletModalProvider>
+      </WalletProvider>
+    </ConnectionProvider>
```
Comment (lorisleiva) [inline mint-ui-example/pages/index.js:69]: nit: could we indent this properly?

**Review by lorisleiva — CHANGES_REQUESTED** [review]

File: getting-started-react-cra5/src/App.js (hunk 1 above)
Comment (MarkSackerberg) [reply getting-started-react-cra5/src/App.js]: looks like I commited a in progress version before i decided to use the connect-wallet example.
reverted.

File: mint-ui-example/pages/index.js (hunk 2 above)
Comment (MarkSackerberg) [reply mint-ui-example/pages/index.js]: linted the whole file.

**Comment by MarkSackerberg** [comment]
should be all done.
Thank you very much for reviewing!

**Review by lorisleiva — CHANGES_REQUESTED** [review]
A few more comments and we should be good to go.

File: getting-started-react-cra5/package.json [hunk 3]
```diff
@@ -3,7 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@metaplex-foundation/js": "0.17.0",
+    "@metaplex-foundation/js": "0.17.4",
+    "@solana/wallet-adapter-react": "^0.15.23",
```
[the remark this answers is not in the document]
Comment (MarkSackerberg) [reply getting-started-react-cra5/package.json]: True. reverted.

File: mint-ui-example/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@metaplex-foundation/js": "0.17.6",
+    "@solana/wallet-adapter-base": "^0.9.17",
+    "@solana/wallet-adapter-react": "^0.15.19",
+    "@solana/wallet-adapter-react-ui": "^0.9.17",
+    "@solana/wallet-adapter-wallets": "^0.19.0",
+    "@solana/web3.js": "^1.63.1",
+    "Buffer": "^0.0.0",
```
[the remark this answers is not in the document]
Comment (MarkSackerberg) [reply mint-ui-example/package.json]: Also correct, this was a result of a test and not required at the end. Removed.

**Comment by MarkSackerberg** [comment]
@lorisleiva from my point of view merging this should be fine.
I will add additional functions (LUT, Mint multiple NFTs at once...) in a future revision, but i would like to merge this first so that we can reference it already.

**Review by lorisleiva — APPROVED** [review]

**Comment by lorisleiva** [comment]
Thank you! 🥳

**Pull request merged**